aten/src/ATen/core/ivalue.h

```diff
@@ -811,6 +891,17 @@ inline bool IValue::isSameIdentity(IValue& rhs) {
         && this->payload.as_intrusive_ptr == rhs.payload.as_intrusive_ptr;
   }
 }
-
-
 } // namespace c10
+
```

Comment:
Same for hashing: it would be better to not expose this on IValue's public API.

Both changes can be done by specifying additional template arguments to the unordered_map.